datacommons/utils.py

```diff
@@ -57,27 +58,13 @@
 # Batch size for heavyweight queries.
 _QUERY_BATCH_SIZE = 500
 
-# Environment variable names used by the package
-_ENV_VAR_API_KEY = 'DC_API_KEY'         # Name the API key variable
-
 
 # --------------------------- API UTILITY FUNCTIONS ---------------------------
 
 
 def set_api_key(api_key):
-  """ Sets an environment variable :code:`"DC_API_KEY"` to given :code:`api_key`.
-
-  An API key is required to use the Python Client API. This can be provided to
-  the API after importing the library, or set as an environment variable
-  :code:`"DC_API_KEY"`.
-
-  For more details about how to get an API key and provide it to the Python
-  Client API, please visit :ref:`getting_started`
-
-  Args:
-    api_key (:obj:`str`): The api key.
-  """
-  os.environ[_ENV_VAR_API_KEY] = api_key
+  """DEPRECATED FUNCTION--API keys no longer required."""
+  logging.warning('Data Commons has removed the API key requirement. This function will be removed by Dec 1st, 2020.')
```

Comment:
@beets wanted you to review this decision :)

Reply:
Lg. Do we have other docs/codes to remove this as well?

Reply:
Yeah need to update docsite... I think that's it. I went through the api-python repo pretty carefully to remove api key refs.

Reply:
Thanks for calling this out!

I don't think we want to remove this completely. We're allowing you to use it without a key, but if you already have one, we should continue allowing you to set it.